test/python/modules/linear_boltzmann_solvers/transport_td/simple_2g_downscatter_td.cxs

```diff
@@ -0,0 +1,24 @@
+NUM_GROUPS 2
+NUM_MOMENTS 1
+
+SIGMA_T_BEGIN
+0 1.0    # group 0
+1 0.8    # group 1
+SIGMA_T_END
+
+VELOCITY_BEGIN
+0 2.0    # v0
+1 0.5    # v1
+VELOCITY_END
+
+TRANSFER_MOMENTS_BEGIN
+# M_GPRIME_G_VAL ell gprime g value  (ell = 0 only)
+# group 0 -> 0: no self-scatter
+M_GPRIME_G_VAL   0   0   0   0.0
+# group 0 -> 1: downscatter
+M_GPRIME_G_VAL   0   0   1   0.5
```

Comment:
looks like gprime is g_departure and g is g_arrival. if you quiz me out of the blue on this, I would have said the opposite.